components/experimental/src/duration/format.rs

```diff
@@ -532,113 +579,38 @@ impl<'a> FormattedDuration<'a> {
         // 6. Let lf be ! Construct(%ListFormat%, « durationFormat.[[Locale]], lfOpts »).
 
         // Note: the above steps are performed while initializing DurationFormatter.
-        let formatted_list = self.fmt.list.format(parts_list.iter().map(|s| &s.string));
-
-        let mut list_sink = PartSink::new();
-        formatted_list.write_to_parts(&mut list_sink)?;
-
-        // 7. Let strings be a new empty List.
```

Comment:
keep these comments and adjust them with the new implementation

Reply:
unless they are redundant

Reply:
They are redundant; I believe @sffc said that it was acceptable to have multiple part annotations from different formatters in the final output. (Let me know if I misunderstood or misquoted you)

The comments are responsible for stripping list formatter parts information from the output.

We could add a divergence from standard note here. What do you think?

Reply:
It's probably good to move some of the spec line item comments to where they are done in the new code structure. But you don't need a 1-for-1 alignment. The code is spec-compliant so long as the behavior is the same; it doesn't need to be line-for-line the same.